### ISSUE TITLE
Create a better demo page to illustrate use

### ISSUE THREAD

**Issue by KrofDrakula** [dhassaine/reduxless#71]
We currently have a very simplistic example that may not illustrate how things interact and what they're for.

It might be good to create something that better represents a toy app one could build with async actions and routing.

**Comment by dhassaine** [comment]
Agreed. I think we can also use github pages to demonstrate a live demo.

**Comment by dhassaine** [comment]
The existing demo is now live at https://dhassaine.github.io/reduxless/demo/public

**Comment by dhassaine** [comment]
I've started a branch to continue this work: https://github.com/dhassaine/reduxless/tree/demo-revamp
